charts/openshift-metering/templates/presto/_presto_helpers.tpl

```diff
@@ -45,6 +45,37 @@ connector.name=blackhole
 connector.name=memory
 {{ end }}
 
+{{- define "presto-prometheus-catalog-properties" -}}
+{{- if .Values.presto.spec.config.connectors.prometheus.enabled }}
```

Comment:
After taking a look at how we manage other connectors, any "empty" (or unused if I understand correctly) have a `<connector-name>.properties` file, that contains the name of the connector. We could modify the above to be something like this, to conform with the rest of the connectors:

```yaml
{{- define "presto-prometheus-catalog-properties" -}}
connector.name=prometheus
{{- if .Values.presto.spec.config.connectors.prometheus.enabled }}
...
```

wdyt?

Reply:
So that we still `prometheus.properties` file with just `prometheus` in it even when the connector is not enabled? I don't think that is right from a Presto standpoint. If the connector is not enabled I think there would be no file present in `.../etc/catalog/`

![image](https://user-images.githubusercontent.com/4449199/70643053-58040600-1c0e-11ea-8c57-6ead6323205c.png)

Reply:
Hmm, I'm seeing this when I disable the prometheus connector:

```bash
tflannag@localhost operator-metering [bentito-MC-720]
└─ $ ▶ k exec -it presto-coordinator-0 -- /bin/bash
Defaulting container name to presto.
Use 'kubectl describe pod/presto-coordinator-0 -n tflannag' to see all of the containers in this pod.
bash-4.2$ cat /opt/presto/presto-server/etc/
catalog/                                config.properties                       jvm.config                              log.properties                          node.properties                         prometheus-connector-bearer-token-file  
bash-4.2$ cat /opt/presto/presto-server/etc/catalog/
..2019_12_11_18_26_16.752759064/ ..data/                          blackhole.properties             hive.properties                  jmx.properties                   memory.properties                prometheus.properties            tpcds.properties                 tpch.properties                  
bash-4.2$ cat /opt/presto/presto-server/etc/catalog/prometheus.properties 
connector.name=prometheus
prometheus-uri=https://prometheus-k8s.openshift-monitoring.svc:9091/
query-chunk-size-duration=1h
max-query-range-duration=1d
cache-duration=30s
bearer-token-file=/var/run/secrets/kubernetes.io/serviceaccount/tokenbash-4.2$ cat /opt/presto/presto-server/etc/catalog/memory.properties     
connector.name=memory
bash-4.2$ 
bash-4.2$ 
bash-4.2$ cat /opt/presto/presto-server/etc/catalog/memory.properties 
connector.name=memory
bash-4.2$ exit
exit
tflannag@localhost operator-metering [bentito-MC-720]
└─ $ ▶ bat $METERING_CR_FILE
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: custom_config/openshift.yaml
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ apiVersion: metering.openshift.io/v1
   2   │ kind: MeteringConfig
   3   │ metadata:
   4   │   name: "operator-metering"
   5   │ spec:
   6   │   unsupportedFeatures:
   7   │     enableHDFS: true
   8   │ 
   9   │   storage:
  10   │     type: "hive"
  11   │     hive:
  12   │       type: "hdfs"
  13   │       hdfs:
  14   │         namenode: "hdfs-namenode-0.hdfs-namenode:9820"
  15   │ 
  16   │   presto:
  17   │     spec:
  18   │       config:
  19   │         connectors:
  20   │           prometheus:
  21   │             enabled: false
───────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
tflannag@localhost operator-metering [bentito-MC-720]
└─ $ ▶ 
```

Reply:
The `prometheus.properties` file is still being populated despite being disabled. Also, It looks like we may need to add a trailing newline to their `prometheus.properties` file.

Reply:
is this just a result of ocp default being enabled?

Reply:
Added prom conn to metering_config.go so the makefile deploy will track other changes in this PR in: 966820f6